src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/RichTextBoxTests.cs

```diff
@@ -7933,6 +7933,20 @@ protected unsafe override void WndProc(ref Message m)
             }
         }
 
+        [WinFormsFact]
+        public void RichTextBox_OleObject_HandleJunkOleObject()
```

Comment:
The original name was good - it provides information on what path is tested
```suggestion
        public void RichTextBox_RichEditOleCallback_CreateStorage()
```

Reply:
I can revert back.

Reply:
In general, we want to avoid words like "junk" in the code base.